src/graphql/subscriptions.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable */
+// this is an auto generated file. This will be overwritten
+
+export const onCreateLightningTalksPoll = /* GraphQL */ `
+  subscription OnCreateLightningTalksPoll($owner: String) {
+    onCreateLightningTalksPoll(owner: $owner) {
+      id
+      type
+      title
+      description
+      username
+      numberOfVotes
+      createdAt
+      updatedAt
+      owner
+    }
+  }
+`;
+export const onUpdateLightningTalksPoll = /* GraphQL */ `
+  subscription OnUpdateLightningTalksPoll($owner: String) {
+    onUpdateLightningTalksPoll(owner: $owner) {
+      id
+      type
+      title
+      description
+      username
+      numberOfVotes
+      createdAt
+      updatedAt
+      owner
+    }
+  }
+`;
+export const onDeleteLightningTalksPoll = /* GraphQL */ `
+  subscription OnDeleteLightningTalksPoll($owner: String) {
+    onDeleteLightningTalksPoll(owner: $owner) {
+      id
+      type
+      title
+      description
+      username
+      numberOfVotes
+      createdAt
+      updatedAt
+      owner
+    }
+  }
+`;
+export const onCreateVotingRecord = /* GraphQL */ `
+  subscription OnCreateVotingRecord($owner: String) {
+    onCreateVotingRecord(owner: $owner) {
+      id
+      lightningTalkPollID
+      username
+      upvote
+      hasVotedBefore
+      createdAt
+      updatedAt
+      owner
+    }
+  }
+`;
+export const onUpdateVotingRecord = /* GraphQL */ `
+  subscription OnUpdateVotingRecord($owner: String) {
+    onUpdateVotingRecord(owner: $owner) {
+      id
+      lightningTalkPollID
+      username
+      upvote
+      hasVotedBefore
+      createdAt
+      updatedAt
+      owner
+    }
+  }
+`;
+export const onDeleteVotingRecord = /* GraphQL */ `
```

Comment:
What does this `/* GraphQL */` do here? Is it just comments or help on formatting as well?